RESOURCES/FEATURE_FLAGS.md

```diff
@@ -84,6 +84,7 @@ These features flags currently default to True and **will be removed in a future
 
 [//]: # "PLEASE KEEP THE LIST SORTED ALPHABETICALLY"
 
+- AVOID_COLORS_COLLISION
```

Comment:
Is this just for testing, and probably removal before the next version? Or do we expect this to live a while and need a deprecation cycle?